src/SlideDotNet/Services/Drawing/ImageEx.cs

```diff
@@ -36,7 +36,7 @@ public ImageEx(SlidePart sldPart, string blipRelateId)
         /// Returns image bytes.
         /// </summary>
         /// <returns></returns>
-        public async ValueTask<byte[]> GetImageBytes()
```

Comment:
One of the reasons to use `ValueTask` over `Task` is `ValueTask` will a little bit effective if the async code executed synchronously. In our case is returning the result from cache.

Or you had some other concern?